doc/nrf/ug_nrf5340.rst

```diff
@@ -625,6 +623,37 @@ Container for firmware update binaries
 The build system will automatically place both the application core and the network core update binaries (:file:`app_update.bin` and :file:`net_core_app_update.bin`) into a container package named :file:`dfu_application.zip`.
 This container package can be used by update tools to pass both images during the simultaneous update of multiple images.
 
+Debugging
+*********
+
+To debug the application core firmware, you need a single debug session.
+The debug session is set up as described in `Debugging nRF5340 with SES`_ or `Debugging an application`_ with Visual Studio Code.
+
+However, to debug the firmware running on the network core, you need to set up separate debug sessions for each core:
+
+* Application core.
+* Network core.
+
+Perform the following steps to start a debugging session in network core:
+
+1. Set up each session for application and network core as mentioned in `Debugging nRF5340 with SES`_ or `Debugging an application`_ with Visual Studio Code.
+#. Select the appropriate CPU for debugging, either the nRF5340 application core or the nRF5340 network core.
+#. Once both sessions are established, run the code execution on the application core.
+   The startup code will release the ``NETWORK.FORCEOFF`` signal to start the network core and it will allocate the necessary GPIO pins for the network core.
+#. Start code execution on the network core in the other debug session.
+
```

Comment:
Three consecutive notes is not ok. I believe one note only should be enough. Maybe the last one.